workflows/templates/kebechet-job.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: kebechet-job
+  annotations:
+    thoth-station.ninja/template-version: 0.1.1
+  labels:
+    app: thoth
+    component: kebechet-job
+
+spec:
+  templates:
+    - name: kebechet-job
+      inputs:
+        parameters:
+        - name: WEBHOOK_PAYLOAD 
+      container:
+        # Chnage to thoth station kebechet master, repo.
+        image: "quay.io/saisankargochhayat/kebechet:master"
```

Comment:
can you update the thoth-station kebechet image here.
https://quay.io/repository/thoth-station/kebechet-job?tab=tags

Reply:
@harshad16 I dont seem to have access to it. Can you grant me access to the repo on quay?

Reply:
image would be pushed by thoth-ops.
` image: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}/{{inputs.parameters.IMAGE_STREAM_NAMESPACE}}/kebechet:{{inputs.parameters.IMAGE_STREAM_TAG}}"`

Reply:
Please have a look at the example from https://github.com/thoth-station/adviser/blob/170e0d2e4d46a055010d03a5b58c643664e3c06c/openshift/adviser-template.yaml#L99

Reply:
@pacospace @fridex WDYT about this ?

Reply:
I have updated it to use image from openshift `thoth-test-core`